content/recipes/authoring/kata-snippets/submission-tests.md

```diff
@@ -0,0 +1,88 @@
+---
+kind: recipe
+sidebar: authoring
+prev: /recipes/authoring/kata-snippets/sample-tests/
+next: /recipes/authoring/kata-snippets/preloaded/
+---
+
+# Writing Submission Tests
+
+This article contains a set of guidelines for kata authors or translators to create good test suites for their kata. They were collected to help ensure that kata, translations and their test suites are of sufficient quality and users' experience with tests will be as good as possible.
+
+The guidelines should be used by kata authors, translators and reviewers to verify whether new content about to be introduced to Codewars is of sufficient quality. Conformance with them should be verified before a kata or translation is published by its author and approved by a reviewer.
+
+Failure to comply with the below guidelines should be considered an issue to be addressed and reported as such. In case of severe violations, the affected kata may be retired or moved back to beta, or the translation rejected.
+
+NOTE: There are many kinds of kata, and some guidelines might simply not apply to some of them (puzzles, hackmes, kata which mess with internals of the language runtime, and some types of challenges that require special ways of testing). But still, the present guidelines apply at least to some extent for the majority of kata, and if they are followed, kata authors can be sure that their tests will be clear, efficient, reliable, and easy to debug and maintain.
+
+
+## General Guidelines
+
+- **Conform to [General Coding Guidelines](/recipes/authoring/kata-snippets/coding-general/)**: tests are code too, and, as such, should keep up to code quality standards.
```

Comment:
```suggestion
- **Conform to [General Coding Guidelines](/recipes/authoring/kata-snippets/coding-general/)**: tests are code too and as such should keep up to code quality standards.
```

(Am I actually reverting changes I introduced myself...? x) Not even impossible, considering how fond I am of commas... x) )

Reply:
LGTM